misc/peer-store/src/behaviour.rs

```diff
@@ -1,24 +1,8 @@
-use std::{collections::VecDeque, task::Poll};
-
 use libp2p_core::{Multiaddr, PeerId};
 use libp2p_swarm::{dummy, NetworkBehaviour};
 
 use crate::store::Store;
 
-/// Events generated by [`Behaviour`] and emitted back to [`Swarm`](libp2p_swarm::Swarm).
-#[derive(Debug, Clone)]
-pub enum Event<T> {
-    /// The peer's record has been updated.  
-    /// Manually updating a record will always emit this event
-    /// even if it provides no new information.
-    RecordUpdated {
-        /// The peer that has an update.
-        peer: PeerId,
-    },
-    /// Event from the internal store.
-    Store(T),
-}
```

Comment:
The behavior doesn't have any own records or data tracking, so it will itself never generate any events.